src/models/pie.js

```diff
@@ -205,7 +205,9 @@ nv.models.pie = function() {
                 }
 
                 if (donutLabelsOutside) {
-                    labelsArc = d3.svg.arc().outerRadius(arc.outerRadius()).startAngle(startAngle).endAngle(endAngle);
+                    labelsArc = d3.svg.arc().outerRadius(arc.outerRadius());
+                    if (startAngle === false) labelsArc.startAngle(startAngle);
+                    if (endAngle === false) labelsArc.endAngle(endAngle);
```

Comment:
Yes, it's !==. Not sure how it worked before. Sorry about that.

The logic above will fail if the user specifies and angle of zero. Should I change that too?